johnzon-osgi/pom.xml

```diff
@@ -29,6 +29,10 @@
   <name>Johnzon :: Support for OSGI Jaxrs Whiteboard</name>
   <packaging>bundle</packaging>
 
+  <properties>
+    <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
guess it will break the release no?

Reply:
yes the osgi properties were causing a real problem for javadoc. I tried excluding all _package-info.java_ but didn't get it working so decided just to skip javadoc on that module